### ISSUE TITLE
Empty window when dowloading source

### ISSUE THREAD

**Issue by sambo57u** [fjwillemsen/NativeOverleaf#25]
When one triies to download the source zip file an empty window opens. The source actually downloads fine but the opening of the window confuses the user. This on Linux version.

**Comment by fjwillemsen** [comment]
I completely agree, it is very confusing and I have not yet found what causes it - most likely it is Nativefier accidentally interpreting it as an external website that must be opened in a new window. Will look into this ASAP.
